src/core/energy_storage/electric_storage.jl

```diff
@@ -36,6 +36,10 @@ worth factor is used in two different ways, depending on the `maintenance_strate
     - `replace_cost_per_kwh`
     - `inverter_replacement_year`
     - `battery_replacement_year`
+    - `replace_cost_per_kw`
```

Comment:
@toddleif these seem to be repeats of the inputs already listed. My original comment was asking if you could add the new inputs that you have created to this list (the cost_constant ones)